### PR TITLE
some tests clean-ups

### PR DESCRIPTION
**Pull request by wind57**

### CODE REVIEW

File: spring-cloud-kubernetes-fabric8-autoconfig/src/test/java/org/springframework/cloud/kubernetes/Fabric8NotInsideInfoContributorTest.java [hunk 1]
```diff
@@ -65,7 +61,13 @@ public void infoEndpointShouldContainKubernetes() {
 				.exchange().expectStatus().isOk().expectBody(String.class).value(this::validateInfo);
 	}
 
-	// {"kubernetes":{"inside":false}}
+	/**
```
Comment (wind57) [inline spring-cloud-kubernetes-fabric8-autoconfig/src/test/java/org/springframework/cloud/kubernetes/Fabric8NotInsideInfoContributorTest.java:64]: nitpick: easier to read

File: spring-cloud-kubernetes-fabric8-autoconfig/src/test/java/org/springframework/cloud/kubernetes/Fabric8NotInsideInfoContributorTest.java [hunk 2]
```diff
@@ -40,8 +39,6 @@
 
 	public static KubernetesServer server = new KubernetesServer();
 
-	private static KubernetesClient mockClient;
```
Comment (wind57) [inline spring-cloud-kubernetes-fabric8-autoconfig/src/test/java/org/springframework/cloud/kubernetes/Fabric8NotInsideInfoContributorTest.java:43]: not needed...

File: spring-cloud-kubernetes-fabric8-autoconfig/src/test/java/org/springframework/cloud/kubernetes/Fabric8NotInsideHealthIndicatorTest.java [hunk 3]
```diff
@@ -75,7 +81,14 @@ public void healthEndpointShouldContainKubernetes() {
 	}
 
 	/**
-	 * kubernetes={ status=UP, details={ inside=false } }
+	 * <pre>
```
Comment (wind57) [inline spring-cloud-kubernetes-fabric8-autoconfig/src/test/java/org/springframework/cloud/kubernetes/Fabric8NotInsideHealthIndicatorTest.java:84]: easier to read

File: spring-cloud-kubernetes-fabric8-autoconfig/src/test/java/org/springframework/cloud/kubernetes/Fabric8NotInsideHealthIndicatorTest.java [hunk 4]
```diff
@@ -65,6 +65,12 @@ public static void setUpBeforeClass() {
 	@AfterAll
 	public static void afterClass() {
 		server.after();
+		System.clearProperty(Config.KUBERNETES_MASTER_SYSTEM_PROPERTY);
```
Comment (wind57) [inline spring-cloud-kubernetes-fabric8-autoconfig/src/test/java/org/springframework/cloud/kubernetes/Fabric8NotInsideHealthIndicatorTest.java:68]: they are set, but not cleaned up. This does not cause any problems currently, but it enforces a good habit to clean-up

File: spring-cloud-kubernetes-fabric8-autoconfig/src/test/java/org/springframework/cloud/kubernetes/Fabric8AutoConfigurationTests.java [hunk 5]
```diff
@@ -19,9 +19,9 @@
 import io.fabric8.kubernetes.client.Config;
 import io.fabric8.kubernetes.client.KubernetesClient;
 import io.fabric8.kubernetes.server.mock.KubernetesServer;
-import org.junit.BeforeClass;
-import org.junit.ClassRule;
-import org.junit.Test;
```
Comment (wind57) [inline spring-cloud-kubernetes-fabric8-autoconfig/src/test/java/org/springframework/cloud/kubernetes/Fabric8AutoConfigurationTests.java:24]: this is what triggered me on this PR to begin with - it was not using JUnit5, like every other test

**Review by ryanjbaxter — APPROVED** [review]

**Pull request merged**